tests/unit/test_openstack.py

```diff
@@ -1301,3 +1313,17 @@ def test_ssl_expiration_false(self):
     def test_ssl_expiration_true(self):
         base = openstack_core.OpenstackBase()
         self.assertTrue(len(base.apache2_certificates_expiring), 1)
+
+    @utils.create_data_root(
+        {'etc/apache2/sites-enabled/openstack_https_frontend.conf':
+         APACHE2_SSL_CONF_RGW})
+    def test_1974138_apache2_allow_encoded_slashes_true(self):
+        base = openstack_core.OpenstackBase()
+        self.assertTrue(base.apache2_allow_encoded_slashes_on, 1)
```

Comment:
The second argument is a message on error, probably not what you intended. It can be:
```
self.assertTrue(base.apache2_allow_encoded_slashes_on)
```